python/ray/ml/examples/pytorch/torch_image_example.ipynb

```diff
@@ -0,0 +1,941 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "38bbd981",
+   "metadata": {},
+   "source": [
+    "# Hello Image Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad65d385",
+   "metadata": {},
+   "source": [
+    "This tutorial illustrates how to train an image classifier using the [Ray AI Runtime](https://docs.ray.io/en/latest/ray-air/getting-started.html).\n",
+    "\n",
+    "You should be familiar with [PyTorch](https://pytorch.org/) before starting the tutorial. If you need a refresher, read PyTorch's [training a classifier](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html) tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b36fbc6",
+   "metadata": {},
+   "source": [
+    "## Before you begin"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2afbd6e7",
+   "metadata": {},
+   "source": [
+    "* Install the [Ray AI Runtime](https://docs.ray.io/en/latest/ray-air/getting-started.html). You'll need Ray 1.13 later to run this example.\n",
+    "\n",
+    "```\n",
+    "pip instsall 'ray[data,tune]'\n",
+    "```\n",
+    "\n",
+    "* Install `torch` and `torchvision`\n",
+    "\n",
+    "```\n",
+    "pip install torch torchvision\n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af1bfc48",
+   "metadata": {},
+   "source": [
+    "## Load and normalize CIFAR-10"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bcc2bbfe",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-09 14:02:17,195\tINFO services.py:1478 -- View the Ray dashboard at \u001b[1m\u001b[32mhttp://127.0.0.1:8265\u001b[39m\u001b[22m\n",
+      "2022-05-09 14:02:21,404\tWARNING read_api.py:248 -- The number of blocks in this dataset (1) limits its parallelism to 1 concurrent tasks. This is much less than the number of available CPU slots in the cluster. Use `.repartition(n)` to increase the number of dataset blocks.\n",
+      "\u001b[2m\u001b[36m(_prepare_read pid=14274)\u001b[0m 2022-05-09 14:02:21,401\tWARNING torch_datasource.py:53 -- `SimpleTorchDatasource` doesn't support parallel reads. The `parallelism` argument will be ignored.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[2m\u001b[36m(_execute_read_task pid=14274)\u001b[0m Downloading https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz to ./data/cifar-10-python.tar.gz\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "0.0%\u001b[36m(_execute_read_task pid=14274)\u001b[0m \n",
```

Comment:
This progress bar looks really bad-- @xwjiang2010 @clarkzinzow this was the same problem that spotify ran into right?